app/models/equipment_model.rb

```diff
@@ -164,63 +164,42 @@ def document_attributes=(document_attributes)
     end
   end
 
-  def num_reserved(start_date, due_date, source_reservations)
-    # get the number reserved in the source reservations
-    # uses 0 queries, you need to specify the max number of
-    # items yourself
-    max_reserved = 0
+  def num_busy(start_date, due_date, source)
```

Comment:
I like the thought behind the term "busy," but I feel like it's basically 99% there. Maybe "allocated" or something like that? That's worse, but I'd like to use something that fits more with what we're doing. It's annoying because `num_unavailable` doesn't work since (in my mind) that would include deactivated items. We'll figure it out.